http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -324,23 +387,44 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
         ) >>
           F.raiseError[Unit](NodepoolDeletionException(params.nodepoolId))
       _ <- nodepoolQuery.markAsDeleted(params.nodepoolId, ctx.now).transaction
+
     } yield ()
 
   override def deleteAndPollApp(params: DeleteAppParams)(
     implicit ev: ApplicativeAsk[F, AppContext]
   ): F[Unit] =
     for {
       ctx <- ev.ask
-      _ <- logger.info(
-        s"Beginning app deletion for app ${params.appId} in project ${params.googleProject} | trace id: ${ctx.traceId}"
-      )
+
       dbAppOpt <- KubernetesServiceDbQueries.getActiveFullAppByName(params.googleProject, params.appName).transaction
       dbApp <- F.fromOption(dbAppOpt, AppNotFoundException(params.googleProject, params.appName, ctx.traceId))
-      // TODO helm delete app and monitor for deletion
+
+      app = dbApp.app
+      namespaceName = app.appResources.namespace.name
+      dbCluster = dbApp.cluster
+      gkeClusterId = dbCluster.getGkeClusterId
+
+      _ <- logger.info(
```

Comment:
should this be `debug`?

Reply:
I dunno, I added a lot of `info` statements to make it clear what's going on when looking at logs. I feel like it's helpful.

Reply:
I find them helpful as well.